catroid/src/main/java/org/catrobat/catroid/ui/SignInActivity.java

```diff
@@ -95,11 +94,7 @@ public void onStop() {
 	}
 
 	public void onButtonClick(final View view) {
-		ConnectivityManager connectivityManager = (ConnectivityManager) getSystemService(Context.CONNECTIVITY_SERVICE);
-		boolean isNetworkAvailable = connectivityManager != null
-				&& connectivityManager.getActiveNetworkInfo().isConnected();
-
-		if (!isNetworkAvailable) {
+		if (!Utils.isNetworkAvailable(this)) {
```

Comment:
Actually I wanted to remove all these static utility classes, that get their context handed through. The reason why it still exists in `Utils` is because I have not had time to remove it everywhere. But if you think it is better this way, we can definitely find another solution.